templates/pages/admin/inventory/agent.html.twig

```diff
@@ -146,5 +148,24 @@
                 })
             ) }}
         {% endfor %}
+
+        <h2 class="header">{{ __('Enabled tasks') }}</h2>
+
+        {% for modulefield, modulelabel in {
+            'use_module_wake_on_lan': __('Wake on LAN'),
+            'use_module_computer_inventory': __('Computer inventory'),
+            'use_module_esx_remote_inventory': __('ESX remote inventory'),
+            'use_module_network_inventory': __('Network inventory (SNMP)'),
+            'use_module_network_discovery': __('Network discovery (SNMP)'),
+            'use_module_package_deployment': __('Package Deployment'),
+            'use_module_collect_data': __('Collect data'),
+            'use_module_remote_inventory': __('Remote inventory'),
+        } %}
+
+        {% if not item.fields[modulefield] %}
+            {% set html = '<i class="ti ti-check"></i>' %}
+        {% endif %}
+        {{ fields.htmlField('', html, modulelabel) }}
+        {% endfor %}
```

Comment:
1. `html` value will inherit from previous item in loop if condition does not match.
2. Are you sure that you want to display a check on `NOT` condition ?